projects/dotcms-field-elements/src/utils/props/validators/date.ts

```diff
@@ -1,6 +1,6 @@
 import { DotDateSlot } from '../../../models';
 
-const DATE_REGEX = new RegExp('\\d\\d\\d\\d-(0[1-9]|1[012])-(0[1-9]|[12][0-9]|3[01])');
+const DATE_REGEX = new RegExp('^\\d\\d\\d\\d-(0[1-9]|1[012])-(0[1-9]|[12][0-9]|3[01])');
```

Comment:
Why?

Reply:
without the `start with` flag,  `AB2010-10-12` is a valid value